### PR TITLE
skip installing driver if module has been loaded

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/features/gpu.py [hunk 1]
```diff
@@ -177,7 +172,7 @@ def get_supported_driver(self) -> List[ComputeSDK]:
         raise NotImplementedError()
 
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
-        self.gpu_vendor: Set[str] = set()
+        self.gpu_vendor: List[str] = ["nvidia"]
```
Comment (squirrelsc) [inline lisa/features/gpu.py]: What happened, if the vendor is not nvidia?

Comment (LiliDeng) [reply lisa/features/gpu.py]: from current logic, I found all of them use nvidia and assigned in the code, maybe I can move it to azure feature.

Comment (squirrelsc) [reply lisa/features/gpu.py]: Can it be checked from code?

Comment (LiliDeng) [reply lisa/features/gpu.py]: updated, now use whether there is any module used for nvidia pci devices, if no driver installed, the used module is empty.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**